scala/wdl4s/build.sbt

```diff
@@ -18,7 +18,7 @@ val DowngradedSprayV = "1.3.1"
 libraryDependencies ++= Seq(
   "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
   "io.spray" %% "spray-json" % DowngradedSprayV,
-  "org.scalaz" %% "scalaz-core" % "7.2.0",
+  "org.scalaz" %% "scalaz-core" % "7.1.3",
```

Comment:
I told you about this the other day. There's a lot of changes between the 2 versions and having it out of synch with cromwell wasn't pleasant. Also I'm not upgrading cromwell for this effort.